charts/aws-efs-csi-driver/CHANGELOG.md

```diff
@@ -1,8 +1,11 @@
 # Helm chart
 
-# v2.2.1
+# master (unreleased)
```

Comment:
conflict had to be resolved here.

for the efs repo, unlike fsx and ebs, the helm canonical source of helm chart is the release branch. So if a helm chart is edited in master branch it isn't actually released unless the change is backported to the release branch. In separate PRs, I'll backport the "Add controller.volMetricsOptIn for emitting volume metrics" change to release branch and then reconcile release and master branch.

The reason for this is because I found during dynamic provisioning release there was a long time between when helm chart was changed and when we could actually release it so it made sense to branch the helm chart to keep it isolated from master branch. Not sure if it's worth continuing this pattern though